### PR TITLE
Update debian files for debian buster [REVPI-3012]

### PR DESCRIPTION
**Pull request by naruxde**

### CODE REVIEW

**Comment by naruxde** [comment]
Don't tell me something about "fix: Discover gpiochip also on CM4s" does not belongs to this PR. Check out History of project and let's move on.

File: debian/copyright [hunk 1]
```diff
@@ -1,7 +1,11 @@
 Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 
 Files: *
-Copyright: 2021 KUNBUS GmbH
+Copyright: 2021-2022 KUNBUS GmbH
```
Comment (hjokunbus) [inline debian/copyright:4]: Why do the dates differ?

Comment (naruxde) [reply debian/copyright:4]: We did a commit in 2022 so update the software and did not update this while the last release of the software.

**Review by hjokunbus — APPROVED** [review]
LGTM

**Review by nbuchwitz — APPROVED** [review]

**Pull request merged**